draft-chen-cfrg-vdaf-pine.md

```diff
@@ -259,40 +261,46 @@ derived from shares of the gradient.
 Operational parameters for the proof system are summarized below in
 {{pine-flp-param}}.
 
-> XXX Finish the table below. Keep the descriptions short: this is table is
-> meant to be used as a reference, not to fully specify things.
-
 | Parameter             | Type     | Description |
 |:----------------------|:---------|:------------|
-| alpha                 | XXX      | XXX         |
-| num_wr_checks         | XXX      | XXX         |
-| num_wr_successes      | XXX      | XXX         |
-| encoded_sq_norm_bound | XXX      | XXX         |
-| wr_check_bound        | XXX      | XXX         |
-| num_bits_for_sq_norm  | XXX      | XXX         |
-| num_bits_for_wr_check | XXX      | XXX         |
-| chunk_len             | XXX      | XXX         |
+| alpha                 | float    | Parameter in wraparound check that determines the ZK error. The higher `alpha` is, the lower ZK error is. |
+| num_wr_checks         | int      | Number of wraparound checks to run. |
+| num_wr_successes      | int      | Minimum number of wraparound checks that a Client must pass. |
+| encoded_sq_norm_bound | Field    | The square of `l2_norm_bound` encoded into a field element. |
+| wr_check_bound        | Field    | The bound of the range check in wraparound checks. |
+| num_bits_for_sq_norm  | int      | Number of bits to encode the squared L2-norm. |
+| num_bits_for_wr_check | int      | Number of bits to encode the range check in each wraparound check. |
+| chunk_length          | int      | Chunk length used in FLP proof. |
 {: #pine-flp-param title="Operational parameters of the PINE FLP."}
 
 ## Measurement Encoding
 
-> XXX Give an overview of `encode_gradient()` and `encode_wr_checks()`. No need
-> to go into full detail yet. The goal is to give the reader an intuition for
-> what the circuit will do, including the bit checks, norm computation, and
-> wraparound checks. Be sure to emphasize that the wraparound test results are
-> not transmitted: instead the Aggregators derive shares of the tests results
-> by running the wraparound tests themselves.
-
-### Encoding of Floating Point Numbers into Field Elements
+The measurement encoding is done in two stages:
+* `PineValid.encode_gradient(self, measurement: list[float]) -> list[Field]`
```

Comment:
Don't refer to syntax that hasn't been defined yet. Instead, how about adding a forward reference to the section that describes it?
```suggestion
* 
```

Reply:
I added a section `Encoding Gradient {#encode-gradient}` and mentions this function there. I also put floating point encoding and L2-norm check as subsections of it.

Reply:
Works for me